onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -356,65 +356,46 @@ void
 MLASCALL
 MlasRequantizeOutput(
     const int32_t* Input,
+    size_t InputLeadingDimension,
     uint8_t* Output,
+    size_t OutputLeadingDimension,
     const int32_t* Bias,
-    size_t M,
-    size_t N,
     const float* Scale,
     bool PerColumnScale,
-    uint8_t ZeroPoint
+    uint8_t ZeroPoint,
+    size_t StartM,
+    size_t StartN,
+    size_t CountM,
+    size_t CountN
     )
-/*++
-
-Routine Description:
-
-    This routine requantizes the intermediate buffer to the output buffer
-    optionally adding the supplied bias.
-
-Arguments:
-
-    Input - Supplies the input matrix.
-
-    Output - Supplies the output matrix.
-
-    Bias - Supplies the optional bias vector to be added to the input buffer
-        before requantization.
-
-    Buffer - Supplies the output matrix.
-
-    M - Supplies the number of elements of the bias vector and the number of
-        rows in the output matrix.
-
-    N - Supplies the number of columns of the output matrix.
-
-    Scale - Supplies the quantization scale.
-
-    PerColumnScale - Supplies true if the quantization scale has per-column
-        values, else false if a single quantization scale applies to the
-        entire matrix.
-
-    ZeroPoint - Supplies the quantization zero point value.
-
-Return Value:
-
-    None.
-
---*/
 {
     const __m128 PerMatrixScaleVector = PerColumnScale ? _mm_setzero_ps() : _mm_load1_ps(Scale);
     const __m128 MinimumValueVector = _mm_set1_ps(float(0 - ZeroPoint));
     const __m128 MaximumValueVector = _mm_set1_ps(float(255 - ZeroPoint));
     const __m128i ZeroPointVector = _mm_set1_epi32(ZeroPoint);
 
+    if (nullptr != Bias) {
+        Bias += StartN;
+    }
+    if (PerColumnScale) {
+        Scale += StartN;
+    }
+
+    Input += StartM * InputLeadingDimension + StartN;
+    Output += StartM * OutputLeadingDimension + StartN;
+
     //
     // Step through each row of the output matrix.
     //
 
-    while (M-- > 0) {
+    while (CountM-- > 0) {
 
         const int32_t* bias = Bias;
         const float* scale = PerColumnScale ? Scale : nullptr;
-        size_t n = N;
+        size_t n = CountN;
+
+        auto* row_in = Input;
```

Comment:
>row_in [](http://example.com/codeflow?start=14&length=6)

MLAS naming style is different with ort. We can change it later, but it would be better to keep consistent before that.